### PR TITLE
UIINREACH-26: Add tests for UIINREACH-13

### PR DESCRIPTION
**Pull request by mariiaschv**
[UIINREACH-26](https://issues.folio.org/browse/UIINREACH-26)

### CODE REVIEW

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      <RenderActionItem
+        id={idDelete}
+        buttonTextTranslationKey={buttonTextTranslationKeyDelete}
+        onClickHandler={handleClickDelete}
+        onToggle={handleToggle}
+      />,
+      translationsProperties,
+    );
+
+    expect(screen.getByRole('button', { name: /Delete/ })).toBeInTheDocument();
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js:80]: In this case, we render our component twice. It is better not to use the beforeEach method unless all tests require passing the same props.

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/GeneralInformation/GeneralInformation.test.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    )
+  ));
+
+  it('should be rendered', () => {
+    const component = () => renderWithIntl(
+      RenderGeneralInformation(),
+      translationsProperties,
+    );
+
+    expect(component()).toBeDefined();
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/GeneralInformation/GeneralInformation.test.js:72]: This is an additional render of the component, you can assign the render from beforeEach to 'let' variable and apply it here.

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    )
+  ));
+
+  it('should be rendered', () => {
+    const component = () => renderWithIntl(
+      RenderServerConnection(centralServer),
+      translationsProperties,
+    );
+
+    expect(component()).toBeDefined();
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js:41]: This is an additional render of the component, you can assign the render from beforeEach to 'let' variable and apply it here.

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        LOCAL_SERVER_CODE: 'localServerCode',
+        CENTRAL_SERVER_ADDRESS: '',
+        LOAN_TYPE_ID: 'loanTypeId',
+        LOCAL_AGENCIES: 'localAgencies',
+        CENTRAL_SERVER_KEY: 'centralServerKey',
+        CENTRAL_SERVER_SECRET: 'centralServerSecret',
+        LOCAL_SERVER_KEY: 'localServerKey',
+        LOCAL_SERVER_SECRET: 'localServerSecret',
+      },
+    };
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js:76]:
```
const testCentralServer = {
      CENTRAL_SERVER_CONFIGURATION_FIELDS: {
        ...CENTRAL_SERVER_CONFIGURATION_FIELDS,
        CENTRAL_SERVER_ADDRESS: '',
      },
    };
```

File: src/settings/components/common/EntityNotFound/EntityNotFound.test.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  it('should be rendered', () => {
+    const component = (handleBack) => renderWithIntl(
+      <RenderEntityNotFound
+        onBack={handleBack}
+      />,
+      translationsProperties,
+    );
+
+    expect(component(handleClickBack)).toBeDefined();
+  });
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/common/EntityNotFound/EntityNotFound.test.js:50]: This is an additional render of the component, you can assign the render from beforeEach to 'let' variable and apply it here.

File: src/settings/components/common/FirsrMenuCloseButton/FirsrMenuCloseButton.test.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  it('should be rendered', () => {
+    const component = () => renderWithIntl(
+      <RenderFirsrMenuCloseButton
+        onClickHandler={handleClick}
+      />,
+      translationsProperties,
+    );
+
+    expect(component()).toBeDefined();
+  });
```
Comment (Dmytro-Melnyshyn) [inline src/settings/components/common/FirsrMenuCloseButton/FirsrMenuCloseButton.test.js:42]: This is an additional render of the component, you can assign the render from beforeEach to 'let' variable and apply it here.

Comment (mariiaschv) [reply src/settings/components/common/FirsrMenuCloseButton/FirsrMenuCloseButton.test.js]: Fixed it, thanks!

File: src/settings/components/common/EntityNotFound/EntityNotFound.test.js (hunk 5 above)
Comment (mariiaschv) [reply src/settings/components/common/EntityNotFound/EntityNotFound.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js (hunk 4 above)
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js (hunk 3 above)
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ServerConnection/ServerConnection.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/GeneralInformation/GeneralInformation.test.js (hunk 2 above)
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/GeneralInformation/GeneralInformation.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js (hunk 1 above)
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  it('should be rendered', () => {
+    const component = (handleClick) => renderWithIntl(
+      <RenderActionItem
+        onClickHandler={handleClick}
+        onToggle={handleToggle}
+      />,
+      translationsProperties,
+    );
+
+    expect(component(handleClickEdit)).toBeDefined();
```
[the remark this answers is not in the document]
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/components/ActionItem/ActionItem.test.js]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/CentralServersConfigurationView.test.js [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+        showActionMenu
+        paneTitle={handlePaneTitle}
+        onBack={handleBack}
+        onDelete={handleDelete}
+        onEdit={handleEdit}
+      />,
+      translationsProperties,
+    );
+
+    expect(screen.getByText('Actions')).toBeDefined();
```
[the remark this answers is not in the document]
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/CentralServersConfigurationView.test.js:72]: Fixed it, thanks!

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/CentralServersConfigurationView.test.js [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+      <RenderView
+        paneTitle={handlePaneTitle}
+        onBack={handleBack}
+        onDelete={handleDelete}
+        onEdit={handleEdit}
+      />,
+      translationsProperties,
+    );
+
+    expect(component()).toBeDefined();
```
[the remark this answers is not in the document]
Comment (mariiaschv) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationView/CentralServersConfigurationView.test.js]: Fixed it, thanks!

**Review by Godlevskyi — APPROVED** [review]

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**